src/cli/src/common/object_store.rs

```diff
@@ -89,6 +89,10 @@ wrap_with_clap_prefix! {
         region: Option<String>,
         #[doc = "Enable virtual host style for the object store."]
         enable_virtual_host_style: bool = Default::default(),
+        #[doc = "Allow anonymous access (disable credential signing) for testing."]
+        allow_anonymous: bool = Default::default(),
+        #[doc = "Disable config load from environment and files for testing."]
+        disable_config_load: bool = Default::default(),
```

Comment:
It’s better to use the same style as `enable_virtual_host_style`.
```suggestion
        enable_config_load: bool = true,
```

Reply:
> It’s better to use the same style as `enable_virtual_host_style`.

both is directly copy from S3Config 
https://docs.rs/opendal/latest/opendal/services/struct.S3Config.html#structfield.enable_virtual_host_style 
and 
https://docs.rs/opendal/latest/opendal/services/struct.S3Config.html#structfield.disable_config_load so I guess their default value is false&reasonable?